README.md

```diff
@@ -1,2 +1,51 @@
 # ChapPress
-A simple project to model best practices for WordPress development.
+ChapPress is a simple WordPress project initiated by the Web and Interactive Marketing team for Chapman University. It is intended to to model best practices for WordPress development. This repository will host the Wordpress model site with custom child themes and plugins.
+
+## Goals
+This WordPress project is intended to provide:
+- A consistent local development environment
+- Source control using standard Git workflow practices
+- Modern automated tests
+- Documentation for best practices and standards
+
+## Requirements
+- Homebrew (1.2.3)
+  - MySQL (5.7.18)
+  - Wordpress Command Line (1.2.1)
+- PHP (5.6.30)
+- Xcode (8.3.3)
+
+## Installation
+
+  Installation process for the setting up Wordpress on a local environment.
+
+  Create the MySQL database.
+
+  ```
+  mysql -u username -p
+  CREATE DATABASE databasename;
+  GRANT ALL PRIVILEGES ON databasename.* TO "wordpressusername"@"hostname" IDENTIFIED BY "password";
+  FLUSH PRIVILEGES;
+  EXIT
+  ```
+  Download WordPress and run the server.
```

Comment:
Let's move the WP core source into this repo. This is what we've done on Blogs:

- https://github.com/chapmanu/blogs/tree/development/public

We've tried it both ways. With our plan to deploy code using Capistrano, having the code in the repo will be simpler. Updates will be done as described there:

- https://github.com/chapmanu/blogs#to-update-the-wp-core